Source/Engine/Core/Math/VectorInt.h

```diff
@@ -1,7 +0,0 @@
-// Copyright (c) 2012-2021 Wojciech Figat. All rights reserved.
```

Comment:
This breaks API so let's retarget PR into `1.2` branch (`master` will be used for next hotfix update for `1.1`).